src/php/k8s-php-install.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+ZEND_API_NO=$1 # zend representation of the PHP version number
+AGENT_FILEPATH=/newrelic-instrumentation/agent
+DAEMON_FILEPATH=/newrelic-instrumentation/daemon
+INSTALL_LOG=/newrelic-instrumentation/newrelic-install.log
```

Comment:
Should these env var be setup to allow overriding and have a default value for future purposes?

Reply:
This is not something the customer can or should be able to configure. These are predetermined locations where the tarball has been extracted, and the log is for us to diagnose problems installing. The ZEND_API_NO is set by the operator.